src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const Button = styled.button`
+  border: none;
+  padding: 0.75rem 1.5rem;
+  border-radius: 0.3rem;
+  font-weight: 400;
+  cursor: ${({ disabled }) => (disabled ? "not-allowed" : "pointer")};
+  color: ${({ variant, theme }) =>
+    variant === "primary"
+      ? theme.colors.primaryText
+      : theme.colors.secondaryText};
+  background-color: ${({ variant, theme }) =>
+    variant === "primary" ? theme.colors.primary : theme.colors.secondary};
```

Comment:
Czy da się zrobić to jakoś bardziej dynamicznie? Co jeśli tych "wariantów" jest 6? Nie będziemy każdego wariantu robić "na piechotę". Jak to zrobić dynamicznie?